lib/hostapd.config.in

```diff
@@ -70,34 +72,34 @@ CONFIG_EAP_GTC=y
 CONFIG_EAP_TTLS=y
 
 # EAP-SIM for the integrated EAP server
-#CONFIG_EAP_SIM=y
+CONFIG_EAP_SIM=y
```

Comment:
Can we get away with only enabling the values in https://github.com/openwrt/openwrt/blob/openwrt-21.02/package/network/services/hostapd/files/hostapd-full.config ?

That way instead of using our own `hostapd`, we can get away with using OpenWRT `hostapd`'s on OpenWRT platforms.

It may also be worth viewing https://salsa.debian.org/debian/wpa/-/blob/debian/unstable/debian/config/hostapd/linux and seeing if we can use Debian's `hostapd` on Debian platforms, but that's a lot less important, since we don't actually have any devices running `edgesec` on Debian currently.

Reply:
No at this moment. For now it will be only for linux preset. The OpenWRT preset will have to use the packaged config.

hostpad needs to be compiled to support a set of options that enable the EAP-TLS functionality and potentially other EAP methods.